app/models/auction_query.rb

```diff
@@ -47,9 +45,16 @@ def complete_and_successful
       .paid
   end
 
-  def rejected
+  def completed
```

Comment:
I guess we are overthinking it. Let's call it completed since we already have a different notion of over....